conversation_simulator/editor/dialogger.js

```diff
@@ -1321,13 +1321,27 @@ function publish () {
   let errors = Validator.getGraphErrors(graph)
   if (errors.length === 0) {
     var gameState = convertToGameState(graph)
-    let name = getModuleName();
-    dtml.dtml.publishDialog(name, version, publisherKey, gameState, function(result){ if (result.ok) { alert("The dialog has been published"); } else alert("There was an error. Please try again");  } );
+    let name = gameState.Setup.Name
+    console.log(name)
+    dtml.dtml.publishDialog(name, version, publisherKey, gameState, function (data) {
+      console.log(data)
+    })
   }
- else
- {
-      alert('Errors in project. Use the \'Validate\' option to find errors.')
```

Comment:
Please keep this line as well, if there are validation errors this code path was silently failing